src/postprocessor.rs

```diff
@@ -59,18 +59,13 @@ pub trait Postprocessor: Send + Sync {
 
 pub fn lookup_with_config(config: &PostprocessorConfig) -> Result<Box<dyn Postprocessor>> {
     match config.name.as_str() {
+        "compress" => Ok(Box::new(Compress::from_config(config.config.as_ref())?)),
         "separate" => Ok(Box::new(separate::Separate::from_config(&config.config)?)),
         "base64" => Ok(Box::new(Base64::default())),
-        "gzip" => Ok(Box::new(Gzip::default())),
```

Comment:
i don't see how removing all those nice short-hands helps users. Now people need to type the very tedious:

```json
{
  "name": "compress",
  "config": {
    "algorithm": "gzip"
  }
}
```

instead of just `"gzip"`.

Reply:
Do we want to also consider adding `elementwise` vs `stream` compression with making the `compress` processor require a config?

Reply:
As we discussed this morning, it reduces the surface we expose as a core, making it simpler, in favor of moving short-hands into userland with definable pre and post processors giving users the option to add syntax sugar to any degree they wish.

Regarding stream compression, so far all compressors have been element wise compression the PR doesn't look at or consider changing this but it'd be a nice ticket to add and consdier

Reply:
Got it.

Lets also have a ticket for defining/creating pre/postprocessors in troy and then i'm good :)

Reply:
Done: 
https://github.com/tremor-rs/tremor-runtime/issues/1541

and for completeness

https://github.com/tremor-rs/tremor-runtime/issues/1540